package.json

```diff
@@ -49,8 +49,9 @@
     "build:styleguide": "styleguidist build",
     "eject": "react-scripts-ts eject",
     "lint": "tslint --project tsconfig.json --config tslint.json && tsc --noEmit --project tsconfig.json",
-    "pretest": "npm run lint",
-    "prebuild": "rimraf dist/** && rimraf build/**",
+    "lint:prod": "tslint --project tsconfig.json --config tslint.prod.json && tsc --noEmit --project tsconfig.json",
+    "pretest": "npm run lint:prod",
+    "prebuild": "rimraf dist/** && rimraf build/** && npm run test",
```

Comment:
In the current CI configuration `npm run test` will be called before `npm run build` as well. Please remove it either here or in `.travis.yml` otherwise it will be called twice in CI.